server/lib/nl/fulfillment/base.py

```diff
@@ -249,3 +253,38 @@ def _svg_or_topic_to_svs(state: PopulateState, sv: str,
     return charts
 
   return []
+
+
+def open_topics_ordered(svs: List[str]) -> List[str]:
+  opened_svs = []
+  sv_set = set()
+  for rank, sv in enumerate(svs):
```

Comment:
is `svs` expected to be ordered in some way to have `rank` make sense? If so, please add a comment to that effect.

Reply:
Added comments